CONTRIBUTING.md

```diff
@@ -404,8 +404,10 @@ We are very strict about `major` changes in the `@biomejs/biome` package. To bet
 
 The description of the changeset should follow the these guidelines:
 
-- Use the past tense, e.g. "Added new feature", "Fixed edge case".
-- If you fixed a bug, please add the link to the issue, e.g. "Fixed edge case [#4444]()".
+- Use the past tense when describing what you did, e.g. "Added new feature", "Fixed edge case".
+- Use the present tense when describing Biome behaviour, e.g. "Biome now supports ...". 
+- If you fixed a bug, please add the link to the issue, e.g. "Fixed [#4444](https://github.com/biomejs/biome/issues/4444)".
+- If you reference a rule, please add the link to the rule on the website, e.g. "Added the rule [`useAwesomeThing`](https://biomejs.dev/linter/rules/use-awesome-thing/)" (even if the website isn't updated yet, the URL is pretty predictable...).
```

Comment:
Can you do reword the phrase to include assist actions too?

Reply:
Thanks!